src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx

```diff
@@ -42,6 +42,11 @@ export class ArtworkDetailsAdditionalInfo extends React.Component<
     return (
       <StackableBorderBox pb={3}>
         <Box>
+          <Box>
+            <Sans size="2" weight="medium">
+              Additional info
+            </Sans>
+          </Box>
```

Comment:
Any reason this has to be wrapped in a box?

Reply:
Without box - it was trying to put it inline. I could have given Sans a display property but thought that it is more descriptive to put it as a separate box to make it easier to move around and mark that it's a separate header element.